### PR TITLE
CNDE-1913: Preprocess service and proc changes for F_STD_PAGE_CASE

### PR DESCRIPTION
**Pull request by johneipe**
## Notes

Preprocess code changes to accomodate attributes for the Fact table - F_STD_PAGE_CASE whichcontaining linking keys for Page Builder STD/HIV investigations.

## JIRA

- **Related story**: [CNDE-1913](https://cdc-nbs.atlassian.net/browse/CNDE-1913)

## Checklist

- [x] PR focuses on a single story.
- [ ] New unit tests added and ensured they pass.
- [x] Service has been tested in local and it works as expected.
- [ ] Documentation has been updated for this code change (if needed).
- [ ] Code follows the Java Coding Conventions (https://www.oracle.com/java/technologies/javase/codeconventions-programmingpractices.html).

## Types of changes

What types of changes does this PR introduces?

- [ ] Bugfix
- [x] New feature
- [ ] Breaking change

## Testing

- [ ] Does this PR has >90% code coverage?
- [ ] Is the screenshot attached for code coverage?
- [ ] Does the `gradle build` pass in your local? 
- [ ] Is the `gradle build` logs attached?

### CODE REVIEW

File: liquibase-service/src/main/resources/db/rdb_modern/tables/007-create_nrt_investigation-005.sql
Comment (sveselev) [inline liquibase-service/src/main/resources/db/rdb_modern/tables/007-create_nrt_investigation-005.sql:1]: Please add a changeset for this script to `db.rdb_modern.changelog-16.1.yaml`.

File: liquibase-service/src/main/resources/db/odse/routines/010-sp_investigation_event-002.sql [hunk 1]
```diff
@@ -767,12 +784,30 @@ BEGIN
...
                  LEFT JOIN
-             (SELECT DISTINCT act_uid       AS                                                 nac_page_case_uid,
-                              MAX(last_chg_time) AS                                                  nac_last_chg_time,
-                              MIN(add_time)      AS                                                 nac_add_time,
-                              MAX(CASE WHEN type_cd = 'PerAsReporterOfPHC' THEN entity_uid END) person_as_reporter_uid,
-                              MAX(CASE WHEN type_cd = 'HospOfADT' THEN entity_uid END)          hospital_uid,
-                              MAX(CASE WHEN type_cd = 'OrgAsClinicOfPHC' THEN entity_uid END)   ordering_facility_uid
+             (SELECT DISTINCT
+                act_uid  AS  nac_page_case_uid,
+                MAX(last_chg_time) AS                                                  nac_last_chg_time,
```
Comment (upasanapattnaik-eq) [inline liquibase-service/src/main/resources/db/odse/routines/010-sp_investigation_event-002.sql:789]: Can we validate that the min/max for the fact table is the same across all the entities? Is it possible that they might have different add or last change times?

File: liquibase-service/src/main/resources/db/changelog/db.rdb_modern.changelog-16.1.yaml [hunk 2]
```diff
@@ -557,4 +557,11 @@ databaseChangeLog:
       changes:
         - sqlFile:
             path: 004-v_getobstxt-001.sql
+            splitStatements: false
+  - changeSet:
+      id: 75
```
Comment (sveselev) [inline liquibase-service/src/main/resources/db/changelog/db.rdb_modern.changelog-16.1.yaml:562]: Conflicting change. The latest id in main branch is 79. Please merge main to this branch.

File: liquibase-service/src/main/resources/db/rdb_modern/tables/007-create_nrt_investigation-001.sql
Comment (sveselev) [inline liquibase-service/src/main/resources/db/rdb_modern/tables/007-create_nrt_investigation-001.sql:1]: 👍🏻

**Review by sveselev — APPROVED** [review]
LGTM!

File: liquibase-service/src/main/resources/db/odse/routines/010-sp_investigation_event-002.sql (hunk 1 above)
Comment (johneipe) [reply liquibase-service/src/main/resources/db/odse/routines/010-sp_investigation_event-002.sql:794]: as discussed will keep the condition as is and proceed.

**Pull request merged**